### ISSUE TITLE
[Question] NodeJS free npm replacement

### ISSUE THREAD

**Issue by Shotman** [voltpkg/volt#48]
Hello,
I'm currently looking for solutions to acheive frontend only  stuff without the need of npm/node at all. I've seen swc to do all the js compiling (tough the rust cli is still wip) grass to do scss compiling, still wip too. So my question is could volt be a total replacement to npm without the need to install node at all on my system ?
Thank you

**Comment by suptejas** [comment]
Hey @Shotman, although this might be something for the future, not planning to implement it as of now. 

It would require developing, maintaining and providing support for multiple tools which is not very feasible (as of now at-least 😁).

Thanks for reaching out and I hope I was able to help 🙂

**Comment by Shotman** [comment]
So if I understood correctly under the hood volt will still require nodejs to be installed on the system and will hook to it while haveing clever caching/simlink features to be more performant than npm/yarn etc ?

**Comment by suptejas** [comment]
That's correct. Volt will be dependent on NodeJS, however we might offer a simple way to install it through the CLI itself. (maybe a `y/n` prompt to install NodeJS)
